config/jobs/kubernetes/sig-aws/eks/k8s-aws-eks-presets.yaml

```diff
@@ -1,23 +1,16 @@
 presets:
 - env:
-  # URL to download 'kubectl', required for 'kubectl' calls to EKS (https://docs.aws.amazon.com/eks/latest/userguide/getting-started.html)
-  # TODO: use upstream 'kubectl'
-  - name: AWS_K8S_TESTER_EKS_KUBECTL_DOWNLOAD_URL
-    value: https://amazon-eks.s3-us-west-2.amazonaws.com/1.11.5/2018-12-06/bin/linux/amd64/kubectl
-  # URL to download 'aws-iam-authenticator', required for 'kubectl' calls to EKS (https://docs.aws.amazon.com/eks/latest/userguide/getting-started.html)
-  - name: AWS_K8S_TESTER_EKS_AWS_IAM_AUTHENTICATOR_DOWNLOAD_URL
-    value: https://amazon-eks.s3-us-west-2.amazonaws.com/1.11.5/2018-12-06/bin/linux/amd64/aws-iam-authenticator
-  # AWS test account credential mounted path, required for AWS API call
+  # Credentials for using AWS test account 607362164682.
```

Comment:
will leave for a follow-up, but perhaps the intention of who should be using this account for what kind of testing? :^)

Reply:
I'm soon planning to organize all aws testing account creds at a central place under `jobs/sig-aws/`. Working with @krzyzacy to get the prow secrets right. I'll clarify it in follow-up PR.

Reply:
See https://github.com/kubernetes/test-infra/pull/10886